jobfunnel/glassdoor.py

```diff
@@ -107,12 +107,12 @@ def get_search_url(self, method='get'):
 
         # get the location id for search location
         location_response = \
-            post(location_url, headers=self.location_headers, data=data).json()
+            self.s.post(location_url, headers=self.location_headers, data=data).json()
 
-        if method is 'get':
+        if method == 'get':
```

Comment:
Could you change this in `indeed.py` aswell?  I didn't know there was a difference between `is` and `==`.  Thanks for noticing.